homeassistant/components/sensor/srp_energy.py

```diff
@@ -0,0 +1,159 @@
+"""
+Platform for retrieving energy data from SRP.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/energy.srp/
```

Comment:
The page name after the period should match the platform module name.

Reply:
fixed